sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/scan/ZapScanner.java

```diff
@@ -92,9 +92,9 @@ public void scan() throws ZapWrapperRuntimeException {
             addReplacerRulesForHeaders();
 
             /* ZAP setup with access to target */
+            importClientCertificate();
```

Comment:
Please add a comment explaining that the order of the calls is important and especially that the client certificate needs to be loaded at the very first.